.circleci/config.yml

```diff
@@ -700,3 +700,9 @@ workflows:
           arch: amd64
           requires:
             - Create version file
+      - Build Linux:
+          name: Build Linux arm64
+          resource_class: arm.medium
+          arch: arm64
+          requires:
+            - Create version file
```

Comment:
Was this job missing? How?? 😳

Reply:
I added it to the `Build and test` workflow, so the arm version is builded on pull requests. It is present in the others workflows.

Reply:
Do we need it, then? I think the build step in build-and-test is just to check for Go compiler errors. No need to worry about OS/arch specifics here.

Reply:
I was going to remove it. It was for generating a build to test on my local machine.
Thinking about it, would you mind if I replace the amd build with arm build in this workflow? So PR builds gives artifacts compatible with our local mac architecture.

Reply:
Sound great!